custom-recipes/googlesheets-append/recipe.json

```diff
@@ -124,6 +124,13 @@
             ],
             "mandatory": true,
             "defaultValue": "USER_ENTERED"
+        },
```

Comment:
Is it possible to change the order? For now, "Show advanced parameters" is the last parameter. I will expect "Sheet name (legacy)" to appears/hide below, not above.

Reply:
But when you click on the recipe of a legacy project, that box appears just above the preferred new selector, with the description "Please use the selector below"

Reply:
I've jut understand what you mean: the help text is not clear, we should replace it by "Legacy, please prefer using the Sheet name selector below."

Reply:
Done in [a1e89a5](https://github.com/dataiku/dss-plugin-googlesheets/pull/9/commits/a1e89a564acc3b406c65b141692f98a5b69e1b7a)